pytition/petition/views.py

```diff
@@ -1530,7 +1530,14 @@ def slug_show_petition(request, orgslugname=None, username=None, petitionname=No
         except SlugModel.DoesNotExist:
             raise Http404(_("Sorry, we are not able to find this petition"))
         petition = slug.petition
-    sign_form = SignatureForm(petition=petition)
+
+    initial = None if not pytitionuser else {
+        'first_name': pytitionuser.user.first_name,
+        'last_name': pytitionuser.user.last_name,
+        'email': pytitionuser.user.email
+    }
+
+    sign_form = SignatureForm(petition=petition, initial=initial)
```

Comment:
This looks great!
Could you add one unit test for this new feature?
Just check that if a user is logged in, its personal information are displayed in the HTML form and that
```python
form.is_bound == True
```
?

Reply:
yep !